storybook/stories/cv-inline-notification-story.js

```diff
@@ -25,7 +25,7 @@ const preKnobs = {
     type: text,
     config: [
       'subtitle',
-      'Lorem ipsum dolor sit amet, consectetur adipisicing elit, seed do eiusmod tempor incididunt ut labore et dolore magna aliqua.',
+      'Lorem ipsum dolor sit amet, <a>consectetur adipisicing elit</a>, seed do eiusmod tempor <strong>incididunt ut labore</strong> et dolore magna aliqua.',
```

Comment:
Add `href="#"` to examples so the mouse cursor looks correct.

Reply:
```suggestion
      'Lorem ipsum dolor sit amet, <a href="#">consectetur adipisicing elit</a>, seed do eiusmod tempor <strong>incididunt ut labore</strong> et dolore magna aliqua.',
```